module/Olcs/src/Controller/Messages/LicenceCloseConversationController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Olcs\Controller\Messages;
+
+use Common\Controller\Interfaces\ToggleAwareInterface;
+use Common\FeatureToggle;
+use Common\Service\Helper\FlashMessengerHelperService;
+use Common\Service\Helper\FormHelperService;
+use Common\Service\Script\ScriptFactory;
+use Common\Service\Table\TableFactory;
+use Dvsa\Olcs\Transfer\Command\Messaging\Close;
+use Laminas\Http\Response;
+use Laminas\View\HelperPluginManager;
+use Laminas\View\Model\ViewModel;
+use Olcs\Controller\AbstractController;
+use Olcs\Data\Mapper\Task;
+
+class LicenceCloseConversationController extends AbstractController implements ToggleAwareInterface
+{
+    protected array $toggleConfig = [
+        'default' => [FeatureToggle::MESSAGING],
+    ];
+    private FlashMessengerHelperService $flashMessengerHelperService;
+
+    public function __construct(
+        ScriptFactory               $scriptFactory,
+        FormHelperService           $formHelper,
+        TableFactory                $tableFactory,
+        HelperPluginManager         $viewHelperManager,
+        FlashMessengerHelperService $flashMessengerHelperService
+    )
+    {
+        parent::__construct($scriptFactory, $formHelper, $tableFactory, $viewHelperManager);
+
+        $this->flashMessengerHelperService = $flashMessengerHelperService;
+    }
+
+    /**
+     * @return ViewModel|Response
+     */
+    public function confirmAction()
+    {
+        $form = $this->getForm('CloseConversation');
```

Comment:
We should use the FQCN for the Close Conversation::Class form class here.

Reply:
When I searched for usage the first one I saw was for strings, didn't notice use of FQCN which is definitely better so gladly!